lib/ex.coffee

```diff
@@ -74,8 +74,13 @@ class Ex
   quit: ->
     atom.workspace.getActivePane().destroyActiveItem()
 
+  quitAll: ->
```

Comment:
so wy when I type :quitall the editor show a message that this command does not exit?